wiki/en/en-Central-Servers.md

```diff
@@ -9,7 +9,7 @@ permalink: "/wiki/Central-Servers"
 
 Because there is a limit of 150 servers per central server, you need to select which central server you want to advertise your server on. Select a genre if you intend to limit players to that style (players can then see your server in their genre list). 
 
-When you start a public server, you will see a message saying whether your server has been successfully listed or not. If not, and you leave your server running, it will re-try its attempts to register until a free slot becomes available.  
+When you start a public server, you will see a message saying whether your server has been successfully listed or not. If not, and you leave your server running, it will reattempt to register until a free slot becomes available.  
```

Comment:
"Keep trying"... keep it simple :)

Reply:
Yes, it does sound more natural that way. Done.